Simplified/AppInfrastructure/NYPLConfiguration+Color.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  NYPLConfiguration+Color.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2021-09-16.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+private enum ColorAsset: String {
+  case defaultBackground
+  case defaultText
+  case downloadBackground
+  case action
+  case deleteAction
+  case fieldBackground
+  case fieldBorder
+  case fieldText
+  case progressBarBackground
+}
+
+@objc extension NYPLConfiguration {
+  static var defaultBackgroundColor: UIColor {
+    if #available(iOS 11.0, *),
+       let color = UIColor(named: ColorAsset.defaultBackground.rawValue) {
+      return color
+    }
+
+    return .white
+  }
+  
+  static var defaultTextColor: UIColor {
+    if #available(iOS 11.0, *),
+       let color = UIColor(named: ColorAsset.defaultText.rawValue) {
+      return color
+    }
+
+    return .black
+  }
+
+  static var downloadBackgroundColor: UIColor {
+    if #available(iOS 11.0, *),
+       let color = UIColor(named: ColorAsset.downloadBackground.rawValue) {
+      return color
+    }
+
+    return .lightGray
+  }
+  
+  static var actionColor: UIColor {
+    if #available(iOS 11.0, *),
+       let color = UIColor(named: ColorAsset.action.rawValue) {
+      return color
+    }
+
+    return .systemBlue
+  }
+  
+  static var deleteActionColor: UIColor {
+    if #available(iOS 11.0, *),
+       let color = UIColor(named: ColorAsset.deleteAction.rawValue) {
+      return color
+    }
+
+    return .systemRed
+  }
+  
+  static var fieldBackgroundColor: UIColor {
```

Comment:
what do these "fieldXYZ" properties refer to? I was thinking input fields but I'm not entirely sure...

Reply:
It is supposed to be used in places where the background is black but not 100% black? Like the background of the first part in `BookDetailView`, but we are using `UIBlurEffect` in there instead of a background color.
The `fieldText` is like above but also for disabled buttons/text.
I named it like its mentioned in Figma ([link](https://www.figma.com/file/SbU22GePd2pXUQ5yrauF15/Dark-Mode?node-id=219%3A71)), so feel free if you have any suggestion on the naming.

Reply:
so if I have a input text field, the box is supposed to be this color?